### PR TITLE
[PL] Translation of String lesson

### PR DESCRIPTION
**Pull request by Koziolek**
In scope of issue #202

@kdebowski have fun :)

### CODE REVIEW

**Comment by nscyclone** [comment]
You may use automatic TOC generation as was described [here](https://github.com/doomspork/elixir-school/pull/352#issuecomment-218850807).

File: pl/lessons/basics/strings.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+iex> string = <<104,101,108,108,111>>
+"hello"
+```
+
+>Uwaga: używając `<< >>` określamy, iż liczby w środku kompilator powinien traktować jak bajty.
+
+## Listy znaków
+
+Wewnętrznie Elixir przechowuje ciągi znaków jako sekwencja bajtów, a nie tablice znaków, posiada jednak osobny typ do
+ reprezentowania listy znaków. Różnica polega na tym, że ciągi znaków tworzymy z użycie podwójnego cudzysłowu `"`, a
```
Comment (kdebowski) [inline pl/lessons/basics/strings.md]: "z użycie" -> "z użyciem"

File: pl/lessons/basics/strings.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+Zwraca listę ciągów znaków, będącą wynikiem podziału ciągu według podanego wzorca. 
+
+```elixir
+iex> String.split("Hello World", " ")
+["Hello", "World"]
+```
+
+## Ćwiczenia
+
+Let's just get in action with two simple exercises to demostrate we are ready to go with Strings!
```
Comment (kdebowski) [inline pl/lessons/basics/strings.md]: English text to remove

File: pl/lessons/basics/strings.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+  def sort_string(string) do
+    string
+    |> String.downcase
+    |> String.graphemes
+    |> Enum.sort
+  end
+end
+```
+
+Przyjrzyjmy się najpierw funkcji `anagrams?/2`. Na początek sprawdzamy, czy parametry są binarny, czy też nie. W
```
Comment (kdebowski) [inline pl/lessons/basics/strings.md]: binarny -> binarne

**Comment by kdebowski** [comment]
@Koziolek I added comments.

**Comment by Koziolek** [comment]
Fixed and merged.

**Pull request merged**